index.html

```diff
@@ -1,74 +1,68 @@
-                <li class="navigation__item"><a href="#" class="navigation__link"><span>03</span>Popular food</a></li>
-                <li class="navigation__item"><a href="#" class="navigation__link"><span>04</span>Food Stories</a></li>
-                <li class="navigation__item"><a href="#" class="navigation__link"><span>05</span>Food now</a></li>
-            </ul>
-        </nav>
-    </div>
-    <header class="header">
-        <div class="header__logo-box">
-            <img src="img/logo-white.png" alt="Logo" class="header__logo" />
-        </div>
-        <div class="header__text-box">
-            <h1 class="heading-primary">
-                <span class="heading-primary--main">Our Food</span>
-                <span class="heading-primary--sub">Is where food happens</span>
-            </h1>
-            <a href="#" class="btn btn--white btn--animated">Discover our food</a>
-        </div>
-    </header>
-    <main>
-        <section class="section-about">
-            <div class="u-center-text u-margin-botton-big">
-                <h2 class="heading-secondary u-margin-botton-big">
-                    FOOD MENU
-                </h2>
+    <body>
+        <header>
+            <nav>
+                <ul>
+                    <li><a href="#"><span>01</span>Food Menu</a></li>
+                    <li><a href="#"><span>02</span>Our Options</a></li>
+                    <li><a href="#"><span>03</span>Popular food</a></li>
+                    <li><a href="#"><span>04</span>Food Stories</a></li>
+                    <li><a href="#"><span>05</span>Food now</a></li>
+                </ul>
+            </nav>
+        </header>
+        <header>
+            <div>
+                <img src="img/logo-white.png" alt="Logo"/>
             </div>
-            <div class="row">
-                <div class="col-1-of-2">
-                    <h3 class="heading-tertiary u-margin-botton-small">Delicious Food</h3>
-                    <p class="paragraph">
-                        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque, dicta voluptas? Totam iure
-                        deserunt, quas reprehenderit recusandae ullam quo! Nobis incidunt architecto quasi explicabo
-                        aperiam alias suscipit necessitatibus temporibus odio!
-                    </p>
-                    <h3 class="heading-tertiary u-margin-botton-small">Different Food</h3>
-                    <p class="paragraph">
-                        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Itaque, dicta voluptas? Totam iure
-                        deserunt, quas reprehenderit recusandae ullam quo! Nobis.
-                    </p>
-
-                    <a href="#" class="btn-text">Learn more &rarr;</a>
+            <div>
+                <h1>
+                    <span>Our Food</span>
+                    <span>Is where food happens</span>
```

Comment:
Creo que podrias modificar "Is where happens" por algo que tu creas mas conveniente.

Reply:
listo